src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -52,6 +55,14 @@ public boolean isExit() {
         return exit;
     }
 
+    public boolean isUserUpdated() {
+        return userUpdated;
+    }
+
+    public boolean hasItemChanged() {
+        return dataToUser == "";
```

Comment:
I'm not sure if this will work as for Strings, we should use the `equals` method.

Perhaps `dataToUser.equals("")` would be better?